xilem/src/view/flex.rs

```diff
@@ -131,69 +131,580 @@ where
         &self,
         view_state: &mut Self::ViewState,
         id_path: &[xilem_core::ViewId],
-        message: xilem_core::DynMessage,
+        message: DynMessage,
         app_state: &mut State,
-    ) -> xilem_core::MessageResult<Action> {
+    ) -> MessageResult<Action> {
         self.sequence
             .seq_message(view_state, id_path, message, app_state)
     }
 }
 
-struct FlexSplice<'f, 'w> {
-    ix: usize,
-    element: &'f mut WidgetMut<'w, widget::Flex>,
-    scratch: AppendVec<Pod<Box<dyn Widget>>>,
+pub enum FlexElement {
+    // Avoid making the enum massive for the spacer cases by boxing
+    Child(Box<Pod<Box<dyn Widget>>>, FlexParams),
```

Comment:
I think `FlexElement` is only transitory, so it doesn't matter if the spacer case is huge.
That is, we're never storing a `Vec<FlexElement>` for very long.

Reply:
Well actually I had it before without double boxing, this is your code from #235, but I'm also fine without it.